package.json

```diff
@@ -55,7 +56,6 @@
     "mocha": "3.4.2",
     "ngrok": "2.2.10",
     "node-watch": "0.5.4",
-    "semver": "5.3.0",
```

Comment:
Sure thing. That comment doesn't apply here because this will run by someone who's installed `zapier-platform-cli` via `npm` (on `zapier upload` and `zapier push`).

`devDependencies` are only installed on an explicit `npm install` inside a `zapier-platform-cli` folder, whereas `dependencies` are installed with a `npm install -g zapier-platform-cli` (`-g` being irrelevant here). Does that make sense?